### ISSUE TITLE
Suggestion: Config option to detect only movement input and stop emote

### ISSUE THREAD

**Issue by NeuTraLZero** [Wet-Boys/LethalEmotesAPI#94]
That way it should probably not break emotes that move you around by default

**Comment by ToastedOven** [comment]
Yeah this is on the backlog, but pretty low priority cause you can just tap c (or whatever you bound the wheel to) to cancel your emotes at any time

**Comment by Nikushimi7** [comment]
it would be very good if emotions were stopped by movement, or at least if the server config could influence this

**Comment by ToastedOven** [comment]
> it would be very good if emotions were stopped by movement, or at least if the server config could influence this

Never adding a feature to let the server influence this, if people want to move and emote they should be allowed to

**Comment by Nikushimi7** [comment]
> > it would be very good if emotions were stopped by movement, or at least if the server config could influence this
> 
> Never adding a feature to let the server influence this, if people want to move and emote they should be allowed to

It's a pity, but I respect your position
